Add some new pages to addon docs

Hexagonl opened the pull request:
I made a draft PR for now because I'd like for @Hans5958 to double check what I've made so far and ensure I'm on the right path. I'll continue making more pages and put the commits for those in this draft PR, to reduce cluttering the repository's PR history.

I'll also propose some guideline additions to the [addon docs wiki](https://github.com/ScratchAddons/website-v2/wiki/Addon-Docs) in a separate issue/PR, which will be based off of stuff I've encountered while making the 60fps addon doc.

### Current added addon docs
- 60fps
- Paint costume by default

Hans5958 commented:
Thank you! I guess I have few suggestions.

1. Instead of "[Issue #6860](https://github.com/ScratchAddons/ScratchAddons/issues/6860)", I prefer "[#6860](https://github.com/ScratchAddons/ScratchAddons/issues/6860)". For that kind of usage I would prefer this instead.

   > - The addon might be marked as dangerous to curb projects that require this addon on the Scratch website ([#6860](https://github.com/ScratchAddons/ScratchAddons/issues/6860)).

   Although I think "issue #6860" (with a little I) should be used just so the general users know that it refers to an issue, but I'm not sure if this is really needed.

2. I think in TurboWarp, it is mentioned as "TurboWarp Addons", with a capital Addons.
3. I would want to see the usage of the footnotes, especially the Trivia which I think you should cite. It works similary like GitHub. For example, like this[^some-cite-name] and this.[^1]

[^1]: It doesn't have to be a number, but you can.
[^some-cite-name]: Hello! This should still appear first even though it is written last on Markdown.

   > ```
   > For example, like this[^some-cite-name] and this.[^1]
   >
   > [^1]: It doesn't have to be a number, but you can.
   > [^some-cite-name]: Hello! This should still appear first even though it is written last on Markdown.
   > ```

WorldLanguages commented:
Nice work!

WorldLanguages commented on content/addons/60fps.md:
```diff
@@ -0,0 +1,61 @@
+- The addon's functionality is only enabled whenever the user toggles it on by holding `alt` and clicking the green flag. The addon's functionality is toggled off every time the page is opened/refreshed.
+- The addon works in both the project page and the editor.
+- By default, the addon (when toggled on) sets the project's framerate to 60 FPS. This value can be changed in the addon's settings to an integer ranging from 31 to 240.
+- To prevent abuse and cheating in cloud-based multiplayer games, the addon does not work whenever there are cloud variables in a shared project.
```
Really?

WorldLanguages replied:
I don't think we check whether there are cloud variables or not.

mxmou replied:
> @Hexagonl Was this removed (or the statement checked?)

It hasn't been removed. The statement is definitely incorrect - there are no references to cloud variables in the addon's code.

WorldLanguages replied:
```suggestion
```

Hexagonl replied:
> @Hexagonl Was this removed (or the statement checked?)

I verified it myself and I did remove it in my latest commit

WorldLanguages commented:
@Hans5958 Please review again if you have some time.

I don't think we should be strict here, as long as there's not any false statements being added to the docs, any page is better than no page

Hans5958 commented on content/addons/paint-by-default.md:
```diff
@@ -0,0 +1,63 @@
+
+## Related
+
+- Original pull request: [PR #3199](https://github.com/ScratchAddons/ScratchAddons/pull/3199)
```
My preference is either...

```suggestion
- [Original pull request](https://github.com/ScratchAddons/ScratchAddons/pull/3199)
```

...or...


```suggestion
- [Original pull request (#3199)](https://github.com/ScratchAddons/ScratchAddons/pull/3199)
```

Also, as stated before, the term "issue" and "PR" is not needed.

Hans5958 reviewed (approved):
To be fair, I'm pretty fine to merge this since the beginning. Problem is that this is still a draft, so it is up to him if he wants this to be merged.

WorldLanguages commented:
Before merging see: https://github.com/ScratchAddons/website-v2/pull/459#discussion_r1703332505

Hexagonl commented:
I fixed my earlier mistake of the cloud variables in the 60fps addon, made the requested changes, and also added an additional reference in the 60fps addon. I also marked as ready to review
